tutor/src/screens/teacher-dashboard/add-menu.jsx

```diff
@@ -1,9 +1,22 @@
 import { React } from 'vendor';
 import { Dropdown } from 'react-bootstrap';
 import { map, partial } from 'lodash';
+import styled from 'styled-components';
 import { autobind } from 'core-decorators';
 import Router from '../../helpers/router';
 import CourseGroupingLabel from '../../components/course-grouping-label';
+import { colors } from 'theme';
+
+const StyledMenuContainer = styled.div`
+  & hr {
+    margin: 0.5rem 1rem;
+    border-top: 2px solid ${colors.neutral.pale};
+  }
+  /* Overriding the color from 'tutor-plan-sidebar-type' */
+  & a[data-assignment-type='template'] {
+    color: ${colors.link} !important;
+  }
```

Comment:
I just read something in the styled-components docs that suggest using `&&&` to work around specificity issues, would that work where?
https://styled-components.com/docs/faqs#how-can-i-override-styles-with-higher-specificity

Reply:
that works and something to really keep in mind. thanks!